great-idea/css/index.css

```diff
@@ -62,3 +62,59 @@ h1, h2, h3, h4, h5 {
     letter-spacing: 1px;
     margin-bottom: 15px;
 }
+
+a:hover {
+	color: lightgrey;
+}
+
+ header {
+	display: flex;
+	justify-content: space-around;
+	border: 1px solid orange;
+}
+
+header nav {
+	display: flex;
+	text-decoration: none;
+  color: grey;
+	justify-content: space-between;
+	align-items: center;
+
+}
+
+header nav a {
+	display: flex;
```

Comment:
Remember that flex is only needed if the selector is of a block type. An `a` tag would be inline.